src/main/java/org/opensearch/security/user/User.java

```diff
@@ -67,7 +67,7 @@
      * roles == backend_roles
      */
     private final Set<String> roles = new HashSet<String>();
-    private final Set<String> securityRoles = new HashSet<String>();
+    private final Set<String> openDistroSecurityRoles = new HashSet<String>();
```

Comment:
Any reason to change `securityRoles ` to `openDistroSecurityRoles `

Reply:
I think this is for to be compatible with the ODFE when doing serializing.

Reply:
Java serialization uses reflection and changing member class variable makes it incompatible between ODFE and OS versions. In the long term, it can be renamed, but will require custom implementation of the `User` class serialization. For now keeping the name makes it compatible.